sys-libs/musl/musl-1.2.4.ebuild

```diff
@@ -121,7 +109,7 @@ src_compile() {
 	just_headers && return 0
 
 	emake
-	if [[ ${CATEGORY} != cross-* ]] ; then
+	if is_crosspkg ; then
```

Comment:
It seems like here and on line 143 you meant `! is_crosspkg` since the original is `[[ ${CATEGORY} != cross-* ]]`

Reply:
yep, thanks!